governance/sponsors_partners.md

```diff
@@ -0,0 +1,110 @@
+To state this another way, the only way for a Partner to influence the project is by
+actively contributing to the open development of the project,
+in equal terms to any other member of the community of Contributors and Council Members.
+Merely using Project Software in institutional context does not allow an entity to become an Institutional Partner.
+Financial gifts do not enable an entity to become an Institutional Partner,
+to be recognized for financial gifts see {ref}`gov_sponsors`.
+Once an institution becomes eligible for Institutional Partnership,
+the RV Council must nominate and approve the Partnership.
+
+If at some point an existing Institutional Partner stops having any contributing employees,
+then a one year grace period commences.
+If at the end of this one year period they continue not to have any contributing employees,
+then their Institutional Partnership will lapse, and resuming it will require going through the normal process for new Partnerships.
+
+An Institutional Partner is free to pursue funding for their work on The Project through any legal means.
+This could involve a non-profit organization raising money from private foundations and donors
+or a for-profit company building proprietary products and services that leverage Project Software and Services.
+Funding acquired by Institutional Partners to work on The Project is called Institutional Funding.
+However, no funding obtained by an Institutional Partner can override the RV Council.
+If a Partner has funding to do ArviZ work and the Council decides to not pursue that work as a project,
+the Partner is free to pursue it on their own.
+However in this situation, that part of the Partner’s work will not be under the ArviZ umbrella
+and cannot use the Project trademarks in a way that suggests a formal relationship.
+
+Institutional Partner benefits are:
+
+(gov_partner_benefits)=
+* All {ref}`sponsor benefits <gov_sponsor_benefits>`
+* Ability to acknowledge their own funding sources on the ArviZ websites
+* Ability to influence the project through the participation of their Institutional Contributors
+  or Council Members.
+
+The minimum amount of paid maintainer time needed to be considered for partnership is
+roughly in line with the sponsorship levels:
+
+* 2 person-months/yr of paid work time for one or more ArviZ contributors to any ArviZ activity
+
+## Historical funding acknowledgement
+All current and previous sponsors, partners, and any other entities and individuals who provided $5,000 or more of financial or in-kind support
+will be acknowledged in the {ref}`historical_funding` page.
+This page will include relevant details of support (dates, amounts, names, and purpose);
+no logos will be used on this page.
+The rationale for the $5,000 minimum level is to keep the amount of work maintaining the page reasonable;
+the level is the equivalent of, e.g., one GSoC or a person-week’s worth of engineering time in a Western country,
```

Comment:
The western country portion gives me pause because it can read that contributions are less valuable depending on where the contributor lives, maybe we just leave that out just state a threshold

Reply:
Good point, I'll delete that